charts/s1-agent/values.yaml

```diff
@@ -30,6 +30,7 @@ configuration:
       fips_enabled: # to enable/disable FIPS mode, set to 'true'/'false'
     helper:
       log_level: "" # info, error, warning, debug, trace (defaults to 'info')
+      client_timeout: # client timeout. (seconds, default=120)
```

Comment:
Should we specify the default here?
I mean:
`client_timeout: 120`

Reply:
I don't think so, we don't do it with any of the settings, check log_level above